src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/IncludeRewriter.cs

```diff
@@ -27,19 +28,17 @@ public override Expression VisitSqlRoot(SqlRootExpression expression, object con
                 return expression;
             }
 
-            bool containsInclude = false;
+            bool containsInclude = expression.TableExpressions.AsEnumerable().Where(e => e.SearchParameterQueryGenerator is IncludeQueryGenerator).Any();
```

Comment:
BTW, expressions like this could be simplified to 
`.Any(e => e.SearchParameterQueryGenerator is IncludeQueryGenerator)`

Reply:
Cool! Thanks. Done.